### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by gooseisgoose** [3kh0/ext-remover#865] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
why does nothing here work

**Comment by Aka-but-nice** [comment, marked as answer]
Yes... they does? Either you cannot follow instructions correctly, or you cannot follow instructions correctly. Either way, that doesn't proclaim the exploit not working.

**Reply by gooseisgoose** [reply]
yah I'm pretty stupid. so your right.

**Reply by Aka-but-nice** [reply]
yr'oue**
